test/lib/mutation_source_test.cc

```diff
@@ -964,7 +964,7 @@ void test_mutation_reader_fragments_have_monotonic_positions(tests::reader_concu
     for_each_mutation([&semaphore, &populate] (const mutation& m) {
         auto ms = populate(m.schema(), {m}, gc_clock::now());
 
-        auto rd = ms.make_reader(m.schema(), semaphore.make_permit());
+        auto rd = ms.make_fragment_v1_stream(m.schema(), semaphore.make_permit());
```

Comment:
These tests should either be converted or removed. We don't need a test suite to run through the new simplified converter.